core/src/main/java/co/aikar/commands/CommandManager.java

```diff
@@ -344,7 +349,32 @@ public void notifyLocaleChange(I issuer, Locale oldLocale, Locale newLocale) {
         });
     }
 
+    public Locale setPlayerLocale(IT player, Locale locale) {
+        I commandIssuer = getCommandIssuer(player);
+        Optional<UUID> uniqueId = commandIssuer.getUniqueId();
+
+        if (uniqueId.isPresent()) {
+            Locale old = issuersLocale.put(uniqueId.get(), locale);
+            if (!Objects.equals(old, locale)) {
+                this.notifyLocaleChange(commandIssuer, old, locale);
+            }
+
+            return old;
+        }
+
+        throw new IllegalArgumentException("The argument " + player + " doesn't implement getUniqueId. We can only " +
```

Comment:
so we can remove this, and note that im trying to avoid game terminology in -core